join.html

```diff
@@ -48,10 +48,59 @@ <h1>SAIL</h1>
         <div class="plate">
           <div class="tile">
-            <p><b>Software Engineer, Cybersecurity</b> (<a href="https://bu.silkroad.com/epostings/index.cfm?fuseaction=app.jobinfo&jobid=301544">apply online</a>)</p>
+            <p><b>Join Our Team</b></p>
             <p>Software engineers at the Software &amp; Application Innovation Lab (SAIL) contribute to a variety of cutting-edge computational and data-driven research efforts while developing their skills in the latest technologies within a professional software engineering environment. The projects undertaken by SAIL software engineers have tangible and direct impacts on academic research, public service, and industry partners in healthcare, finance, and technology.</p>
-            <p>As a cybersecurity expert, you will work within an agile team building prototypes and production-quality open source libraries, applications, and systems that provide stronger security benefits in domains such as trustworthy distributed computing (blockchains and secure multi-party computation), software-defined networking/clouds, and systems security at the OS, firmware, and hardware layers. This role provides the flexibility to identify and prioritize efforts and to shape the long-term vision of our research agenda. In addition to working with other professional software engineers, you have the opportunity to collaborate with researchers at Boston University and beyond who are nationally respected as leaders in their corresponding areas of research. Additionally, we expect you to take advantage of opportunities to develop new skills in almost every project and enjoy tackling diverse challenges in a flexible work environment, and to mentor software engineer interns.</p>
-            <p>SAIL is housed within the Hariri Institute for Computing, which catalyzes, enables, and supports interdisciplinary research efforts across Boston University.</p>
+            <p><b>Current Openings</b></p>
+            <ul>
+              <li><a href="#full-stack">Software Engineer, Full-Stack</a></li>
+              <li><a href="#cybersecurity">Software Engineer, Cybersecurity</a></li>
+            </ul>
+            <p>All team members at SAIL are engaged in identifying efforts and shaping the long-term vision of our research agenda. In addition to working with other professional software engineers, you have the opportunity to collaborate with researchers at Boston University and beyond who are nationally respected as leaders in their corresponding areas of research. Additionally, you can develop new skills in almost every project, tackle diverse challenges in a flexible work environment, and mentor software engineer interns.</p>
+            <p>SAIL is housed within the Hariri Institute for Computing, which catalyzes, enables, and supports interdisciplinary research efforts across Boston University. Our team consists of ten full-time staff members and 10-15 interns each semester.</p>
+            <hr>
```

Comment:
I tried changing the CSS to make this divider look better. I wanted the border color to be` #777777` and the border width to be around 0.1px, but it didn't work whether I made a new class in the CSS or changed the `<hr>` style in this file.

Reply:
```
hr {
    border-bottom-color: #777;
    border-top: none;
}
```
Subpixel sizing should be avoided, since every browser deals with it in a different way. The HTML/CSS on that page isn't very clean though, it's difficult to position that \<hr> properly. Beezoo didn't want to use Bootstrap in favor of their custom CSS "framework", but I think we should just go back to the basic Bootstrap grid.